### ISSUE TITLE
Fail gracefully if no file is provided

### ISSUE THREAD

**Issue by richgilbank** [darcyclarke/rawkit#37]
Currently when no filename is provided, it blows up - would be nice to validate that it's been given an arg and provide a helpful error message, or (better yet), check for `package.json` in the current dir and read it's `main` script to use as the default behaviour. 

>  rawkit

```
/usr/local/bin/rawkit
Devtools URL: chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:9229/06fa8646-bdf8-4dc2-a05d-3c5f33894c80
/usr/local/bin/rawkit
events.js:182
      throw er; // Unhandled 'error' event
      ^
Error: listen EADDRINUSE :::1337
```

**Comment by darcyclarke** [comment]
PR? 😛

**Comment by darcyclarke** [comment]
@richgilbank this will be out with the next release. Also fixing up the need for an extension server by using github pages 👍

**Comment by darcyclarke** [comment]
This should be fixed now as of the `v1.2` release.
